docs/gettingStarted/setupGenericScheme.md

```diff
@@ -0,0 +1,217 @@
+      }).send()
+
+  let genericScheme = await genericSchemeDeployedContract
+  
+  // Log Address of new instance to use in next step while registering the scheme to DAO
+  console.log(`Deployed new GenericScheme instance at ${genericScheme.options.address}`)
+
+  // Following are example values, Please change appropriately
+  // Refer https://daostack.zendesk.com/hc/en-us/sections/360000535638-Genesis-Protocol
+  const voteParams = {
+        "boostedVotePeriodLimit": 345600,
+        "daoBountyConst": 10,
+        "minimumDaoBountyGWei": 150000000000,
+        "queuedVotePeriodLimit": 2592000,
+        "queuedVoteRequiredPercentage": 50,
+        "preBoostedVotePeriodLimit": 86400,
+        "proposingRepRewardGwei": 50000000000,
+        "quietEndingPeriod": 172800,
+        "thresholdConst": 1200,
+        "voteOnBehalf": "0x0000000000000000000000000000000000000000",
+        "votersReputationLossRatio": 4,
+        "activationTime": 0
+      }
+
+  // Get address from https://github.com/daostack/migration/blob/master/migration.json
+  const votingMachineAddress = "0xaddress-of-VotingMachine-of-DAO-on-given-network"
+
+  // For eg if you want this Generic Scheme to enable DAO to interact with Bounties Network
+  // then targetContract would be the address of Bounties Network's respective contract
+  const targetContractAddress = "0xaddress-of-contract-this-will-interact-with"
+
+  const avatar = "0xaddres-of-DAO"
+
+  // paramHash will be useful in later step so lets log it
+  const paramHash = genericScheme.methods.initialize(
+      avatar,
+      voteParams,
+      votingMachineAddress,
+      targetContractAddress
+      ).call()
+
+  console.log(paramHash)
+
+  genericScheme.methods.initialize(
+        avatar,
+        voteParams,
+        votingMachineAddress,
+        targetContractAddress
+      ).send()
+```
+
+#### Submit new proposal to Scheme Registrar via Alchemy UI
+
+  1. On Alchmey landing page choose the DAO to which you wish to register the scheme
+  2. Visit DAO's `Home` page and choose `Scheme Registrar`
+  3. Click `New Proposal`, this will open a popup
+  4. Select `Add Scheme` on the popup sidebar (on left)
+  5. Give appropriate title, description, url which describes the proposal
+  6. Put the address of Generic Scheme contract
+
+      **NOTE**: Based on above step this would be either address of UGenericScheme (if registering generic scheme for first time or editing params for existing one) or address of GenericScheme instance deployed in previous step (if registering multiple generic scheme to the DAO)
+
+  7. Enter the `paramHash` you got [here](#set-generic-scheme-to-interact-with-your-contract)
+  8. In permissions section choose `Call genericCall on behalf of`
+  9. Submit Proposal and sign off the transaction
+  10. If the DAO agrees then your Generic Scheme with ability to interact with the `targetContract` will be registered to the DAO
+
+## How to get your Generic Scheme shown in Alchemy
```

Comment:
Maybe it's worth to give an example of an existing config json. I know I always understood better after examining a real example rather than just an explanation... :)

I gave this one to the person integrating the bounties network and I think he understood it ^_^
`src/components/Proposal/ProposalSummary/ProposalSummaryDutchX.tsx`

Reply:
Aah I had missed the `ProposalSummary` part. 
Will update and add the example